spec/student_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'spec_helper'
+
+describe Student do
+  before :each do
+    @classroom = Classroom.new('Label')
+    @student = Student.new(22, @classroom)
+  end
+
```

Comment:
- [ ] It would be better if we write test for our class method `classrooms` as we have in the class itself :+1:

Reply:
There is a testfile for the classroom class